arangod/IResearch/IResearchLinkHelper.cpp

```diff
@@ -69,24 +69,17 @@ bool isIgnoredHiddenEnterpriseCollection(std::string const& cName) {
    * This should be fine. Another attempt could be to rewrite the links itself,
    * but more code changes will then be necessary.
    */
-  if (arangodb::ServerState::instance()->isSingleServer()) {
-    if (cName[0] == '_') {
-      if (strncmp(cName.c_str(),
-                  arangodb::StaticStrings::FullLocalPrefix.c_str(),
-                  arangodb::StaticStrings::FullLocalPrefix.size()) == 0 ||
-          strncmp(cName.c_str(),
-                  arangodb::StaticStrings::FullFromPrefix.c_str(),
-                  arangodb::StaticStrings::FullFromPrefix.size()) == 0 ||
-          strncmp(cName.c_str(), arangodb::StaticStrings::FullToPrefix.c_str(),
-                  arangodb::StaticStrings::FullToPrefix.size()) == 0) {
-        LOG_TOPIC("d921b", DEBUG, arangodb::Logger::VIEWS)
-            << "Ignoring link to '" << cName
-            << "'. Will only be initially created via SmartGraphs of a full "
-               "dump of a cluster."
-               "This link is not supposed to be restored in case you dump from "
-               "a cluster and then restore into a single-server instance.";
-        return true;
-      }
+  if (!arangodb::ServerState::instance()->isSingleServer()) {
```

Comment:
The logic here seems reversed, i.e. from `isSingleServer()` to `! isSingleServer()`.
Is that intentional?

Reply:
No